pkg/porter/testdata/porter.yaml

```diff
@@ -1,22 +1,40 @@
 mixins:
-- exec
+  - exec
```

Comment:
I see we also have a new `bundle-with-custom-action.yaml` bundle in testdata, which this now appears to be equivalent to.  Just wanted to check that this is intended.

Reply:
Yeah I don't know what our testdata rules are? Personally I like to keep the testdata close to the test, and not try to keep mega reusable test files.  They do have different data in them (one is missing the customActions section).

If we wanted to reduce the amount of duplicate testdata, I'd vote for a set of rules to follow and sweep in a separate issue because I know that I've done this throughout Porter. 😊